mautic/mautic
# Fix callback mailjet processing If contact has uppercase email address

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

If contact's email address is uppercase ten bounce processing for mailjet does not work. This PR fixed it



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.  Create email address TEST@TESTABC12345XXX.COM
3. Setup mailjet transport with callback processing
4. Send  email
5. Check If email was processed as bounce




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11670"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
